### PR TITLE
Fix import of rides without vehicles, fix ride availability, fixes #3414

### LINKED ISSUE #3414: Info Kiosk Glitch when importing RCT1 saves
OS: Windows 10
Version:0.0.5

When I import some of my RCT1 saves into OpenRCT the Info kiosks will display as a coaster train rather than an info kiosk.

I've included zip files of all the save files that display the kiosks correctly and another one which displays the coaster train

Screenshot:
untitled

[With Info Kiosk Glitch.zip](https://github.com/OpenRCT2/OpenRCT2/files/233555/With.Info.Kiosk.Glitch.zip)
[Works.zip](https://github.com/OpenRCT2/OpenRCT2/files/233557/Works.zip)

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/rct1/S4Importer.cpp [hunk 1]
```diff
@@ -890,6 +889,7 @@ void S4Importer::ImportResearch()
     // gResearchExpectedDay =
     // gResearchExpectedMonth =
 
+	sub_684AC3();
```
Comment (IntelOrca) [inline src/rct1/S4Importer.cpp]: This function resets the progress and stage etc. This needs to be moved up to just after ```research_remove_non_separate_vehicle_types```

**Comment by Gymnasiast** [comment]
Ok, moved it up.

**Comment by IntelOrca** [comment]
I said after ```research_remove_non_separate_vehicle_types```

**Comment by Gymnasiast** [comment]
Done.

**Pull request merged**